lib/mix/lib/releases/assembler.ex

```diff
@@ -497,6 +505,12 @@ defmodule Mix.Releases.Assembler do
       do: :ok
   end
 
+  defp generate_cookie do
+    # 255 is max. atom length, see http://erlang.org/doc/efficiency_guide/advanced.html
+    # Cookies, at least when read from disk, need to be from the printable ASCII range
+    (1..255)|> Enum.map(fn _ -> Enum.random(?\s..?~) end) |> List.to_atom
```

Comment:
http://erlang.org/doc/installation_guide/INSTALL.html talks a bit about openssl. Apparently it is not strictly required for erlang, and `strong_rand_bytes` would be missing then.

I'll make that explicit in the code, and advise people with a key generation snipped using `strong_rand_bytes` assuming most people have that available.